src/main/java/org/folio/linked/data/integration/kafka/listener/SourceRecordDomainEventListener.java

```diff
@@ -57,7 +58,7 @@ private void processRecord(ConsumerRecord<String, SourceRecordDomainEvent> consu
       .map(SourceRecordType::fromValue)
       .orElseThrow();
 
-    if (sourceRecordType == MARC_AUTHORITY) {
+    if (sourceRecordType == MARC_AUTHORITY || sourceRecordType == MARC_BIB) {
```

Comment:
we can completely remove this check aren't we?

Reply:
nope, there are more types in SRS, holdings for example